### PR TITLE
Add suffixes to typos that start with "ap"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -3696,7 +3696,15 @@ aotomatically->automatically
 aotomation->automation
 aovid->avoid
 apach->apache
+apapt->adapt
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:3699]: Could it be `apart` in some cases?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -3776,15 +3813,26 @@ apostrafies->apostrophes
...
+appache->apache
 appar->appear
+apparance->appearance
+apparances->appearances
 apparant->apparent
 apparantly->apparently
+appareance->appearance
+appareances->appearances
 appared->appeared
+apparen->apparent
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:3825]: Perhaps `apparel` too? **L** and **N** are close on Dvorak keyboards, for what it's worth.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:3699]: I couldn't find `apapt->apart`.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:3825]: I couldn't find `apparen->apparel`.

**Pull request merged**